### ISSUE TITLE
Duplicate username not checked on loadup

### ISSUE THREAD

**Issue by 0WN463** [CS2113-AY1819S1-F10-1/main#164]
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<AddressBook isPermAdmin="true">
    <persons>
        <name>John Doe</name>
        <phone isPrivate="false">98765432</phone>
        <email isPrivate="false">johnd@gmail.com</email>
        <address isPrivate="true">311, Clementi Ave 2, #02-25</address>
        <fees isPrivate="true">0.00</fees>
        <duedate>00-00-0000</duedate>
        <tagged>owesMoney</tagged>
        <tagged>friends</tagged>
        <account>
            <username>a</username>
            <password>b</password>
            <privilege>Admin</privilege>
        </account>
    </persons>
    <persons>
        <name>John Dae</name>
        <phone isPrivate="false">98765432</phone>
        <email isPrivate="false">johnd@gmail.com</email>
        <address isPrivate="true">311, Clementi Ave 2, #02-25</address>
        <address isPrivate="true">311, Clementi Ave 2, #02-25</address>
        <fees isPrivate="true">0.00</fees>
        <duedate>00-00-0000</duedate>
        <tagged>owesMoney</tagged>
        <tagged>friends</tagged>
        <account>
            <username>a</username>
            <password>c</password>
            <privilege>Admin</privilege>
        </account>
    </persons>
    <password>pw</password>
</AddressBook>
```

Note both person have username starting with a.
Current behavior: App loads up, logging in with a,b is possible. a,c will result in wrong password
Expected behavior: App fails to load up.

**Comment by 0WN463** [comment]
Solved in #158

**Comment by 0WN463** [comment]
@micklwl you can confirm that this if fixed in the latest jar
